packages/editor/src/components/document-outline/item.js

```diff
@@ -17,8 +12,9 @@ const TableOfContentsItem = ( {
 	children,
 	isValid,
 	level,
-	onClick,
 	path = [],
+	target,
+	close,
```

Comment:
Same question as above?

Reply:
same answer as above, should I rename this?

Reply:
renamed this prop to `closeOutline` https://github.com/WordPress/gutenberg/pull/10815/commits/f0afad5aec0a4429e80212767130e6505d032f8f